src/ios/CDVCamera.h

```diff
@@ -87,19 +91,51 @@ typedef NSUInteger CDVMediaType;
 {}
 
 @property (strong) CDVCameraPicker* pickerController;
-@property (strong) NSMutableDictionary *metadata;
+@property (strong) NSDictionary* metadata;
 @property (strong, nonatomic) CLLocationManager *locationManager;
-@property (strong) NSData* data;
+@property (strong) UIImage* image;
 
 /*
- * getPicture
+ * takePicture
  *
  * arguments:
- *	1: this is the javascript function that will be called with the results, the first parameter passed to the
- *		javascript function is the picture as a Base64 encoded string
- *  2: this is the javascript function to be called if there was an error
+ *  1: success callback (called with either Base64, file URI or native URI)
```

Comment:
These comments seem to be meant to document the javascript side of things, instead of the objective-c side. What's the reason for them being here ?